src/main/java/org/iatoki/judgels/gabriel/blackbox/algorithms/IdentityEvaluator.java

```diff
@@ -0,0 +1,38 @@
+package org.iatoki.judgels.gabriel.blackbox.algorithms;
+
+import com.google.common.collect.ImmutableList;
+import org.apache.commons.io.FilenameUtils;
+import org.iatoki.judgels.FileSystemProvider;
+import org.iatoki.judgels.LocalFileSystemProvider;
+import org.iatoki.judgels.gabriel.blackbox.EvaluationException;
+import org.iatoki.judgels.gabriel.blackbox.EvaluationResult;
+import org.iatoki.judgels.gabriel.blackbox.Evaluator;
+
+import java.io.File;
+import java.io.IOException;
+
+public final class IdentityEvaluator implements Evaluator {
```

Comment:
- [ ] Probably better name it `OutputOnlyEvaluator` since it does do something: unzipping the file